src/Util/ClassSourceManipulator.php

```diff
@@ -123,7 +131,16 @@ public function addEmbeddedEntity(string $propertyName, string $className)
             ),
         ];
 
-        $this->addProperty($propertyName, $annotations);
+        $attributes = [
+            $this->buildAttributeNode(
+                'ORM\\Embedded',
+                [
+                    'class' => new ClassNameValue($className, $typeHint),
+                ]
+            ),
+        ];
+
+        $this->addProperty($propertyName, $annotations, null, $attributes);
```

Comment:
We're passing both the annotations and attributes to `addProperty()`? Won't that cause both to be added? We might need another `$useAttributesForMapping` argument to the `addEmbeddedEntity()` that helps us determine which to create.